repository-data/application/src/main/resources/hcm-content/content/documents/administration/lks/value-lists/categories.yaml

```diff
@@ -0,0 +1,27 @@
+/content/documents/administration/lks/value-lists/categories:
```

Comment:
@thojava Should I remove the categories from application data or keep them in both modules(development and application)?

Reply:
@adaMiclos I don't think that is a good idea to keep that on both, actually that is duplicated because the content of `application` and `development` will be merged on bootstrapping time. 

We should only keep it under the `development` module and we can have some kind of release guidance to acknowledge to admin about that.

Reply:
ok, I'll do that. Thanks !